### ISSUE TITLE
build docker error

### ISSUE THREAD

**Issue by qiushaoxi** [EOSIO/eos#1363]
Step 19/26 : COPY --from=builder /opt/eosio/bin /opt/eosio/bin
COPY failed: stat /var/lib/docker/aufs/mnt/77d84015e0c1b36ede35b28b02861968b83f8e732c9e5e22385ea92cf275cee2/opt/eosio/bin: no such file or directory

**Comment by IcculusC** [comment]
I discovered that the build process inside the `Dockerfile` actually passes an environmental variable to `cmake` that is not respected by the build.  `CMAKE_INSTALL_PREFIX` is overriden in the `installer.cmake` file.  Instead of `/opt/eos` the files are in installed directory that the `installer.cmake` defines.

I was able to fix it by changing the `Dockerfile` to copy the files from the `/eos/build/install/bin` directory to `/opt/eos/bin`.

The `Dockerfile` in `master` may be a little different than `tags/DAWN-2018-02-14`, but I think your problem is the same as mine.
